src/callbacks_step/analysis.jl

```diff
@@ -455,13 +499,43 @@ end
 # some common analysis_integrals
 # to support another analysis integral, you can overload
 # Trixi.analyze, Trixi.pretty_form_utf, Trixi.pretty_form_ascii
-function analyze(quantity, du, u, t, semi::AbstractSemidiscretization)
+function analyze(quantity, du_ode, u_ode, t, semi::AbstractSemidiscretization)
   mesh, equations, solver, cache = mesh_equations_solver_cache(semi)
+
+  du = wrap_array(du_ode, mesh, equations, solver, cache)
+  u = wrap_array(u_ode, mesh, equations, solver, cache)
+
   analyze(quantity, du, u, t, mesh, equations, solver, cache)
 end
-function analyze(quantity, du, u, t, mesh, equations, solver, cache)
-  integrate(quantity, u, mesh, equations, solver, cache, normalize=true)
+
+
+function analyze(quantity, du_ode, u_ode, t, semi::SemidiscretizationCoupled; normalize=true)
```

Comment:
Is this the reason why we had the many places changed where you could specify `normalize` as a keyword argument?

Reply:
Yes, we need the non-normalized errors, so we can add them up and normalize with the sum of the total volumes of the meshes.

Reply:
Would it be possible (without excessive redundancy, just considerable ugliness) to just de-normalize the individual l2_errors and then normalize the total error? I am asking since this would be a much less intrusive approach in terms of the current API. I am not saying it's what we should do, I just wanna see the options we have

Reply:
Should be possible, yes. But many functions already had this normalize option, so we used it and added the keyword argument for those that didn't support it.

Reply:
OK, let's leave this open for now and see if it resolves itself when some of the other comments trigger changes.